docs/src/tutorials/getting_started/getting_started_with_JuMP.jl

```diff
@@ -198,6 +198,12 @@ dual_status(model)
 # Now we know that our solver found an optimal solution, and that it has a
 # primal and a dual solution to query.
 
+# It is good practice in your code to include statements like this so that an
```

Comment:
maybe not even "good practice" but something stronger like "very important" or "imperative"

Reply:
Yeah.

Reply:
Note that assertions can be disabled: https://docs.julialang.org/en/v1/base/base/#Base.@assert

Reply:
Not sure it matters much for docs though